pkg/apiserver/rest/usecase/definition.go

```diff
@@ -318,6 +318,12 @@ func renderDefaultUISchema(apiSchema *openapi3.Schema) []*utils.UIParameter {
 			params = append(params, param)
 		}
 	}
+	sort.Slice(params, func(i, j int) bool {
```

Comment:
Suggested sorting conditions:
1. Check `validate.required`.  It is True, the sort number will be lower.
2. Check `subParameters`. The more subparameters, the larger the sort number.
3. The sort number starts at 100.

Reply:
fixed.